### PR TITLE
Improve Disk Usage

### PR DESCRIPTION
**Pull request by rafaelbreno**

### CODE REVIEW

**Review by briandowns — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,11 +1,10 @@
 name: Push Image
 run-name: Release ${{ github.ref_name }}
 on:
-  release:
-    types: [published]
+  pull_request:
```
Comment (tashima42) [inline .github/workflows/release.yml:4]: Shouldn't you roll back these changes?

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -20,29 +19,60 @@ jobs:
...
     - name: Build and push image
       uses: rancher/ecm-distro-tools/actions/publish-image@master
       with:
         image: hardened-kubernetes
         tag: ${{ github.event.release.tag_name }}
         public-repo: rancher
-        public-username: ${{ env.DOCKER_USERNAME }}
-        public-password: ${{ env.DOCKER_PASSWORD }}
+        public-username: rancher
+        public-password: rancher
```
Comment (tashima42) [inline .github/workflows/release.yml:55]: I don't think this should be hard-coded, and the secrets step should be added back

File: Dockerfile [hunk 3]
```diff
@@ -51,25 +56,28 @@ RUN echo "export GO_LDFLAGS=\"-linkmode=external \
...
     -X k8s.io/client-go/pkg/version.gitTreeState=clean \
     -X k8s.io/client-go/pkg/version.buildDate=\${BUILD_DATE} \
-    \"" >> /usr/local/go/bin/go-build-static-k8s.sh
-RUN echo 'go-build-static.sh -gcflags=-trimpath=${GOPATH}/src/kubernetes -mod=vendor -tags=selinux,osusergo,netgo ${@}' \
+    \"" >> /usr/local/go/bin/go-build-static-k8s.sh && \
+    echo 'go-build-static.sh -gcflags=-trimpath=${GOPATH}/src/kubernetes -mod=vendor -tags=selinux,osusergo,netgo ${@}' \
     >> /usr/local/go/bin/go-build-static-k8s.sh
 RUN chmod -v +x /usr/local/go/bin/go-*.sh
 
+## maybe here
```
Comment (tashima42) [inline Dockerfile:64]:
```suggestion

```

File: Makefile [hunk 4]
```diff
@@ -77,7 +77,6 @@ push-image:
 		--build-arg K3S_ROOT_VERSION=$(K3S_ROOT_VERSION) \
 		--build-arg K8S_TAG=$(shell echo $(TAG) | grep -oP "^v(([0-9]+)\.([0-9]+)\.([0-9]+))") \
 		--tag $(IMAGE) \
-		--push \
```
Comment (tashima42) [inline Makefile:80]: Probably should also be reverted

**Review by tashima42 — CHANGES_REQUESTED** [review]
If this will actually be used for pull-requests, I think you should change the file name, if those are just test changes, a roll back should fix it, everything else LGTM

**Pull request closed without merging**